### PR TITLE
Adding weather icons

### PR DESCRIPTION
**Pull request by yaydogan**
Changes:
* Added weather icons
* Displaying temperature with single decimal digit with degree sign
* Added daily temperature summary to the weather summary text

### CODE REVIEW

**Comment by andrewda** [comment]
:+1: This looks like it should work well.

File: index.html [hunk 1]
```diff
@@ -44,14 +46,19 @@
       <div class="top-right">
         <div class="weather">
           <div class="weather-today">
+            <span class="icon dimmed wi {{currentForcast.wi}}"></span> 
             <span class="tempreture">{{currentForcast.temperature}}&deg;</span>
           </div>
-          <div class="weather-week-descriptor">{{weeklyForcast.summary}}</div>
+ 		  <div class="weather-week-descriptor">
```
Comment (evancohen) [inline index.html]: decrease indentation

**Comment by evancohen** [comment]
Looks good @yaydogan! Fix this and I'll merge it in.

**Comment by yaydogan** [comment]
Fixed indentation.

**Pull request merged**